templates/Project/Etude/TabVoir/Redaction.html.twig

```diff
@@ -386,7 +388,15 @@
                 <td>
                     <a href="{{ path('project_procesverbal_modifier', {'id': etude.pvr.id}) }}">{{ 'suivi.pvr'|trans({}, 'project') }}</a>
                 </td>
-                <td>{{ 'suivi.toutes'|trans({}, 'project') }}</td>
+                <td>
+                    {% if etude.pvr.phases is null %}
```

Comment:
Possible de faire comme le commentaire d'au dessus avec le else dans la boucle. (Plus de détails à https://twig.symfony.com/doc/3.x/tags/for.html#the-else-clause)

Par contre attention au type, `etude.pvr.phases` peut ne pas être null (un tableau ou une ArrayCollection vide). Il eu fallu vérifier que la longueur de phases est égale à 0.(finalement même pas nécessaire grace a la clause else)